packages/office-ui-fabric-react/src/components/FloatingPicker/Suggestions/Suggestions.types.ts

```diff
@@ -83,6 +87,10 @@ export interface ISuggestionsControlProps<T> extends React.Props<any>, ISuggesti
    * Completes the suggestion
    */
   completeSuggestion: () => void;
+  /**
+   * Flag indicates if render suggestions as a div element
+   */
+  renderSuggestionsItemAsDiv?: boolean;
```

Comment:
Does this need to be defined twice since `ISuggestionsControlProps` extends `ISuggestionsCoreProps<T>`

```ts
export interface ISuggestionsControlProps<T> extends React.Props<any>, ISuggestionsCoreProps<T> {
```

Reply:
I also wonder if we'd be better off using `renderAs` pattern here vs. a `boolean`.

Reply:
Nice catch. Fixing. ;)